src/main/java/net/fabricmc/installer/launcher/NativesHelper.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.installer.launcher;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class NativesHelper {
+    private static final String OS_ID = getOS() + "-" + System.getProperty("os.arch");
+    private static final Map<String, String> NATIVES_MAP = getNativesMap();
+
+    private static boolean loaded = false;
+
+    static {
+        System.out.println("OS_ID: " + OS_ID);
+    }
+
+    private static Map<String, String> getNativesMap() {
+        Map<String, String> natives = new HashMap<>();
+
+        natives.put("windows-arm64", "natives/windows-ARM64.dll");
+        natives.put("windows-Win32", "natives/windows-Win32.dll");
+        natives.put("windows-amd64", "natives/windows-x64.dll");
+
+        return natives;
+    }
+
+    private static String getOS() {
+        String osName = System.getProperty("os.name").toLowerCase();
```

Comment:
```suggestion
        String osName = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
```

Needs the proper import ofc